test/e2e/network_segmentation_services.go

```diff
@@ -156,9 +156,11 @@ var _ = Describe("Network Segmentation: services", func() {
 				By("Connect to the UDN service cluster IP from the UDN client pod on the same node")
 				checkConnectionToClusterIPs(f, udnClientPod, udnService, udnServerPod.Name)
 				checkConnectionToNodePort(f, udnClientPod, udnService, &nodes.Items[0], "endpoint node", udnServerPod.Name)
-				// TODO uncomment below as soon as ovnkube-node supports UDN
-				// checkConnectionToNodePort(f, clientPod2, udnService, &nodes.Items[1], "client node", udnServerPod.Name)
-				// checkConnectionToNodePort(f, clientPod2, udnService, &nodes.Items[2], "other node", udnServerPod.Name)
+				// FIXME(dceara): Remove this check when Local Gateway external->service support is implemented.
+				if !IsGatewayModeLocal() {
+					checkConnectionToNodePort(f, udnClientPod, udnService, &nodes.Items[1], "client node", udnServerPod.Name)
```

Comment:
nit: we should probably also add a couple more ingress tests. I'm thinking external IP, and maybe LB IP. Perhaps with ETP local. Don't want to overkill this since we plan on migrating to k8s tests, but would be nice until then to have a little more coverage. Can be done in follow up.